.github/workflows/claude.yml

```
@@ -0,0 +1,163 @@
+name: Claude
+on:
+  pull_request:
+    branches: ["main"]
```

Consider running pull requests into "main" with labeled PRs:

```suggestion
    branches: ["main"]
    types: [ "labeled" ]
```